documentation/docs/components/outputs/command-outputs/nym-node-migrate-help.md

```diff
@@ -1,72 +1,8 @@
 ```sh
```

Comment:
I think that the latest release may not even allow to migrate, that's why this came. out empty